### PR TITLE
feat: add docker compose for app, update node.js in container

### PR DESCRIPTION
**Pull request by tciesek**
Run app using docker-compose:
```sh
docker-compose --env-file .env -f docker-compose.yml -f docker-compose.db.yml up --build
```

### CODE REVIEW

File: docker-compose.db.yml [hunk 1]
```diff
@@ -10,9 +10,14 @@ services:
       POSTGRES_DB: ${API_DB_DATABASE}
     ports:
       - ${API_DB_PORT}:5432
+    volumes:
+      - pg-data:/var/lib/postgresql/data
```
Comment (tciesek) [inline docker-compose.db.yml:14]: Persist database in docker volume (no need to migrate everytime when there are no database changes)

Comment (adrkakol) [reply docker-compose.db.yml:14]: Nice, thanks!

Comment (sjedlikowski) [reply docker-compose.db.yml:14]: We can host DB in a separate RDS instance instead of sharing t2.micro resources between API and DB :bulb:

Comment (tciesek) [reply docker-compose.db.yml:14]: Yes, I agree @sjedlikowski but how does it affect the docker-compose file with db configuration for local development? 

I guess that for production usage we will indeed use postgres provided by Heroku or RDS.

Comment (sjedlikowski) [reply docker-compose.db.yml:14]: Aaaargh, I'm getting blind :-|
I thought that it was in the `Dockerfile`...

**Pull request merged**